documentation/sections/announcing_coffeescript_2.md

```diff
@@ -0,0 +1,82 @@
+- All of the above was achieved with very few [breaking changes from 1.x](http://coffeescript.org/v2/#breaking-changes). Most current CoffeeScript projects should be able to upgrade with little or no refactoring necessary.
+
+CoffeeScript 2 was developed with two primary goals: remove any incompatibilities with JavaScript that might prevent CoffeeScript from being used on a project; and preserve as much backward compatibility as possible. [Install now](http://coffeescript.org/v2/#installation): `npm install -g coffeescript@2`
+
+## ES2015+ Output
+
+From the beginning, CoffeeScript has been described as being “just JavaScript.” And today, JavaScript is ES2015 (well, ES2017). CoffeeScript welcomes the changes in the ES world and we’re happy to stop outputting ES3  syntax for modern CoffeeScript and ES2015+ features.
+
+(Quick history lesson: ECMAScript is the official name of JavaScript, after the ECMA standards body that defines it. ECMAScript version 3, or ES3, was released in 1999 and is the lowest common denominator version of JavaScript that anyone today considers supporting. There was no ES4, and ES5 came out in 2009. What would’ve been ES6 was eventually released as ES2015, a change in naming meant to indicate that future versions would be released every year. When we write “ES2015+”, we mean ES2015, ES2016, ES2017 and future successors.)
+
+Many ES2015 features, such as `=>`, were adopted directly from CoffeeScript and are one-to-one compatible, or very nearly so. This has made outputting many of CoffeeScript’s innovations into ES2015 syntax straightforward: not only does `=>` become `=>`, but `{ a } = obj` becomes `{ a } = obj`, `"a#{b}c"` becomes `` `a${b}c` `` and so on.
+
+The following CoffeeScript features were updated in 2 to output using ES2015+ syntax (or added in 1.11–2, output using modern syntax):
+
+- Modules
+- Classes
+- Async functions, a.k.a. `async`/`await`
+- Bound/arrow functions
+- Function default parameters
+- Function splat/rest parameters
+- Destructuring, for both arrays and objects
+- Object rest/spread properties
+- Interpolated strings/template literals (ES2015 backticked strings)
+- Tagged template literals
+- ES2015’s `for…of` is now available as CoffeeScript’s `for…from` (we already had a `for…of`)
+
+Not all CoffeeScript features were adopted as is; most notably, [default values](http://coffeescript.org/v2/#breaking-changes-default-values) in ES2015 and CoffeeScript 2 are only applied when a variable is `undefined`, not `undefined` or `null` as in CoffeeScript 1; and [classes](http://coffeescript.org/v2/#breaking-changes-classes) have their own differences. See the [breaking changes](http://coffeescript.org/v2/#breaking-changes) for the fine details.
```

Comment:
I found it a bit confusing who was adopting what here at first. Doing something like the following might help:

- `Not all CoffeeScript features were adopted as is` → `Not all CoffeeScript features were adopted by ES2015 100% the same way`
- `default values in ES2015 and CoffeeScript 2` → `default values in ES2015 (and also CoffeeScript 2)`